### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by Mzuha**

### CODE REVIEW

File: src/main/java/mate/academy/spring/dao/impl/UserDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            session = sessionFactory.openSession();
+            transaction = session.beginTransaction();
+            session.save(user);
+            transaction.commit();
+            return user;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Cant add user to db " + user, e);
```
Comment (Daniel1004K) [inline src/main/java/mate/academy/spring/dao/impl/UserDaoImpl.java:35]: maybe create your custom exception

File: src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    @Autowired
+    private UserDao userDao;
```
Comment (Tamara-Paskalova) [inline src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java:13]: The best practice is to use injection through the constructor neither through the field.

**Review by Tamara-Paskalova — APPROVED** [review]

File: pom.xml [hunk 3]
```diff
@@ -17,6 +17,38 @@
...
+            <version>8.0.25</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.5.5.Final</version>
+            <scope>compile</scope>
+        </dependency>
+
```
Comment (KseniiaMakarova) [inline pom.xml:50]:
```suggestion
```

File: src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java (hunk 2 above)
Comment (KseniiaMakarova) [reply src/main/java/mate/academy/spring/service/impl/UserServiceImpl.java:13]: I agree

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**